website/content/docs/app-config/dynamic.mdx

```diff
@@ -106,6 +105,79 @@ $ waypoint config source-set -type=vault -config="auth=aws"
 The documentation for the source settings available to a configuration source
 are alongside their parameters in the plugin documentation.
 
+### Scoped Source Settings
+
+The `waypoint config source-set` command's `-scope` flag will configure a given
+config source to be used within a specific scope. This can be either `global`,
+(the default), `project`, or `app`-scoped. For a given app or project, the config
```

Comment:
Maybe it makes sense to pull out the valid scope targets into a bulleted list for easy reading? something like:

```markdown
- **global**: Source setting applied to all projects and their applications
- **project**: Applies source setting only to *This* project
- **app**: Applies source setting only to *this* application
```

Reply:
Added!